src/templates/components/colours/palette.scss

```diff
@@ -0,0 +1,128 @@
+$guardian-brand: #c70000;
```

Comment:
I've created a ticket for us to use this palette, as we currently don't use it

Reply:
Some of these colour values could use a [Source makeover](https://theguardian.design/2a1e5182b/p/71fb50-colour/b/399c59) too 💅

Reply:
Will add this as a note on the newly created ticket